bundles/mapping/camera-controls-3d/instance.js

```diff
@@ -43,5 +56,37 @@ Oskari.clazz.defineES('Oskari.mapping.cameracontrols3d.instance',
             this.sandbox = null;
             this.started = false;
         }
+        _registerForGuidedTour () {
+            const me = this;
+            function sendRegister () {
+                const requestBuilder = Oskari.requestBuilder('Guidedtour.AddToGuidedTourRequest');
+                if (requestBuilder && me.sandbox.hasHandler('Guidedtour.AddToGuidedTourRequest')) {
+                    const delegate = {
+                        bundleName: me.getName()
+                    };
+                    for (let prop in GUIDED_TOUR_TEMPLATE) {
+                        if (typeof GUIDED_TOUR_TEMPLATE[prop] === 'function') {
+                            delegate[prop] = GUIDED_TOUR_TEMPLATE[prop].bind(me); // bind methods to bundle instance
+                        } else {
+                            delegate[prop] = GUIDED_TOUR_TEMPLATE[prop]; // assign values
+                        }
+                    }
+                    me.sandbox.request(me, requestBuilder(delegate));
+                }
+            }
+
+            function handler (msg) {
+                if (msg.id === 'guidedtour') {
+                    sendRegister();
+                }
+            }
+
+            const tourInstance = me.sandbox.findRegisteredModuleInstance('GuidedTour');
```

Comment:
Could `sandbox.hasHandler('Guidedtour.AddToGuidedTourRequest')` be used here instead of checking for the bundle?